WW 2353 Radio revamp

kevinbarfleur opened the pull request.

kevinbarfleur committed:
test

kevinbarfleur committed:
test

kevinbarfleur committed:
test

kevinbarfleur committed:
test

kevinbarfleur committed:
test

kevinbarfleur committed:
wip

kevinbarfleur committed:
test

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
test

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
test update

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
wip

kevinbarfleur committed:
Update

kevinbarfleur committed:
Clean update

kevinbarfleur committed:
Refactor

AurelieV commented on ww-config.js:
```diff
@@ -1,15 +1,25 @@
     options: {
         autoByContent: true,
-        layout: ['flex', 'grid'],
+        displayAllowedValues: ['flex', 'grid'],
```
ajouter inline-flex et inline-grid

kevinbarfleur committed:
inline-flex et inline-grid

AurelieV committed:
Update for AI

AurelieV committed:
Use a v-for instead of a wwLayout

leo91000 committed:
feat: add AI.md

MarcFabre committed:
Update AI.md

leo91000 committed:
WW-2353: Add form support to ww-radiogroup component

- Add form-specific properties: fieldName, customValidation, validation
- Integrate form functionality using inject pattern
- Update documentation to reflect form-only properties
- Enable radiogroup to work within form containers with validation

leo91000 committed:
WW-2353: Remove reference to ww-radiogroup-item

Component is redundant and has been removed

leo91000 committed:
WW-2353: Mark valueFormula as required in AI.md

Update documentation to help AI understand that valueFormula is required

leo91000 committed:
fix(WW-2353): add isArray to itemElement

leo91000 committed:
chore(WW-2353): Improve AI.md
